server/game/cards/02_SHD/Headhunting.ts

```diff
@@ -0,0 +1,39 @@
+import AbilityHelper from '../../AbilityHelper';
+import { UnitCard } from '../../core/card/CardTypes';
+import { EventCard } from '../../core/card/EventCard';
+import { Trait } from '../../core/Constants';
+
+export default class Headhunting extends EventCard {
+    protected override getImplementationId() {
+        return {
+            id: '5896817672',
+            internalName: 'headhunting',
+        };
+    }
+
+    public override setupCardAbilities() {
+        this.setEventAbility({
+            title: 'Attack with up to 3 units',
+            immediateEffect: AbilityHelper.immediateEffects.sequential([
+                this.buildBountyHunterAttackEffect(),
+                this.buildBountyHunterAttackEffect(),
+                this.buildBountyHunterAttackEffect()
+            ])
+        });
+    }
+
+    private buildBountyHunterAttackEffect() {
+        return AbilityHelper.immediateEffects.selectCard({
+            innerSystem: AbilityHelper.immediateEffects.attack({
+                targetCondition: (card) => !card.isBase(),
+                effects: AbilityHelper.ongoingEffects.conditionalAttackStatBonus(
+                    (attacker: UnitCard) => attacker.hasSomeTrait(Trait.BountyHunter),
```

Comment:
Can we de-anonymize these properties? I.e. something like
cardCondition: (attacker: UnitCard) => attacker.hasSomeTrait(Trait.BountyHunter),
bonus: { power: 2, hp: 0}

If not possible, not a dealbreaker